shared/common-adapters/button.js

```diff
@@ -70,30 +73,27 @@ class Button extends React.Component<Props> {
       containerStyle = collapseStyles([containerStyle, styles.opacity30])
     }
 
-    if (!isMobile && this.props.waiting) {
+    if (this.props.waiting) {
       labelStyle = collapseStyles([labelStyle, styles.opacity0])
     }
 
     containerStyle = collapseStyles([containerStyle, this.props.style])
 
     const onClick = (!this.props.disabled && !this.props.waiting && this.props.onClick) || null
-
-    const whiteSpinner = this.props.type !== 'PrimaryGreenActive'
+    const whiteSpinner = !(this.props.type === 'PrimaryGreenActive' || this.props.type === 'Secondary')
```

Comment:
:sheep:   `!['PrimaryGreenActive', 'Secondary'].includes(this.props.type)`

Reply:
i've found this form to confuse flow so i've avoided it.unsure if newer flow is smarter about this

Reply:
ah np, that's too bad, it's very convenient for whitelists

Reply:
https://flow.org/try/#0C4TwDgpgBAglC8UDkB7AdhJUA+zgHcUkAoYgY3QGdgoA3BZACwgBsWV8TiAzAVzTLAAluijcAFAEMAXLACUUAN7EoUCmkooWEAHTsA5lLnEAvqSHco4+vFvJ0mHLht2kBIguWqJtY2eIWVgDaqBhIADR4hEgAujpCAiy8ACYQlNZynipiGaZAA